test/backend-test/test-real-browser-keyword.js

```diff
@@ -0,0 +1,150 @@
+const test = require("node:test");
+const assert = require("node:assert");
+const { RealBrowserTestHelper, UP } = require("./real-browser-test-helper");
+
+// Initialize the test helper
+const helper = new RealBrowserTestHelper();
+
+// Test content variations with predictable keywords
+const testPages = {
+    withHello: helper.createTestServer("Hello World! This is a test page with sample content."),
+    withoutHello: helper.createTestServer("This is a test page with sample content but no greeting."),
+    withSpaces: helper.createTestServer("Hello    World!\n\n   This is a    test page."),
+    withSpecialChars: helper.createTestServer("Hello World! Special chars: àáâãäå øñü"),
+    withNumeric: helper.createTestServer("Status: 200 OK - Server is running with code 123"),
+    longContent: helper.createTestServer("A".repeat(100) + " This is some long content for truncation testing")
+};
+
+// Predictable keywords for our test content
+const availableKeywords = {
+    found: "Hello", // Present in withHello pages
+    notFound: "Missing", // Not present in any pages
+    partial: "World", // Present in withHello pages
+    numeric: "123", // Present in withNumeric page
+    special: "àáâãäå" // Present in withSpecialChars page
+};
+
+test("Real Browser Monitor Integration Tests", {
+    skip: process.env.CI && !process.env.UPTIME_KUMA_ENABLE_BROWSER_TESTS
```

Comment:
why this?

Reply:
i have removed the condition, as most tests dont have these.